banjo/conf/schema.xml

```diff
@@ -219,7 +219,10 @@
    <field name="additional_contributor" type="text_general" indexed="true" stored="true"/>
    <field name="additional_creator" type="text_general" indexed="true" stored="true"/>
    <field name="access_agreement" type="string" indexed="true" stored="true"/>
-
+   <field name="access_agreement_reason_for_change" type="string" indexed="true" stored="true"/>
+   <field name="access_condition_reason_for_change" type="string" indexed="true" stored="true"/>
+   <field name="commercial_status_reason_for_change" type="string" indexed="true" stored="true"/>
+   <field name="access_comments" type="string" indexed="true" stored="true"/>

```

Comment:
Last time I added another field "access_agreement" and @scoen told me using "string", I guess these four new fields are similar. What's the difference between string and text_general?